backend/framework/qlf/dashboard/bokeh/qagetbias/main.py

```diff
@@ -33,105 +34,32 @@ def load_qa(self):
         nrg = check_ccds['PARAMS']['BIAS_AMP_NORMAL_RANGE']
         wrg = check_ccds['PARAMS']['BIAS_AMP_WARN_RANGE']
         tests = mergedqa['TASKS']['CHECK_CCDs']['PARAMS']
+        refexp=mergedqa['TASKS']['CHECK_CCDs']['PARAMS']['BIAS_AMP_REF']
```

Comment:
This will break after the changes Sarah did https://github.com/desihub/desispec/pull/699 but we can fix it later